src/murrine_style.c

```diff
@@ -1182,6 +1182,22 @@ murrine_style_draw_box (DRAW_ARGS)
 
 		murrine_set_widget_parameters (widget, style, state_type, &params);
 
+		if (murrine_style->trough_use_child_bg && widget->parent && MRN_IS_SCROLLED_WINDOW (widget->parent)) {
+			GdkColor *gcolor = NULL;
+			GtkWidget *child  = gtk_bin_get_child ((GtkBin*) widget->parent);
```

Comment:
> Can you explain what's happening here? Wouldn't `gtk_bin_get_child (widget->parent)` just return `widget`?

Sorry, I misunderstood your question. `widget` is a Scrollbar inside a ScrolledWindow and `widget->parent` is the actual container with the scrolled child inside. I hope its clearer now :)